src/Everscale/Wallet.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <boost/integer.hpp>
+#include <utility>
+
+#include "../PublicKey.h"
+
+#include "Address.h"
+#include "Cell.h"
+#include "CellBuilder.h"
+#include "CellSlice.h"
+
+const uint32_t WALLET_ID = 0x4BA92D8A;
+
+namespace TW::Everscale {
+
+class Wallet {
+public:
+    struct Gift {
+        bool bounce;
+        uint64_t amount;
+        Address::MsgAddressInt destination;
+        uint8_t flags;
+    };
+
+    static constexpr const uint8_t code[] = {
+        0xB5, 0xEE, 0x9C, 0x72, 0x01, 0x01, 0x01, 0x01, 0x00, 0x71, 0x00,
+        0x00, 0xDE, 0xFF, 0x00, 0x20, 0xDD, 0x20, 0x82, 0x01, 0x4C, 0x97,
+        0xBA, 0x21, 0x82, 0x01, 0x33, 0x9C, 0xBA, 0xB1, 0x9F, 0x71, 0xB0,
+        0xED, 0x44, 0xD0, 0xD3, 0x1F, 0xD3, 0x1F, 0x31, 0xD7, 0x0B, 0xFF,
+        0xE3, 0x04, 0xE0, 0xA4, 0xF2, 0x60, 0x83, 0x08, 0xD7, 0x18, 0x20,
+        0xD3, 0x1F, 0xD3, 0x1F, 0xD3, 0x1F, 0xF8, 0x23, 0x13, 0xBB, 0xF2,
+        0x63, 0xED, 0x44, 0xD0, 0xD3, 0x1F, 0xD3, 0x1F, 0xD3, 0xFF, 0xD1,
+        0x51, 0x32, 0xBA, 0xF2, 0xA1, 0x51, 0x44, 0xBA, 0xF2, 0xA2, 0x04,
+        0xF9, 0x01, 0x54, 0x10, 0x55, 0xF9, 0x10, 0xF2, 0xA3, 0xF8, 0x00,
+        0x93, 0x20, 0xD7, 0x4A, 0x96, 0xD3, 0x07, 0xD4, 0x02, 0xFB, 0x00,
+        0xE8, 0xD1, 0x01, 0xA4, 0xC8, 0xCB, 0x1F, 0xCB, 0x1F, 0xCB, 0xFF,
+        0xC9, 0xED, 0x54,
+    };
```

Comment:
Do you have a doc that reference this constant?

Reply:
I've added comment with link to contract but it seems we don't have any doc on this wallet.

Reply:
0x52B2241F491e5937b8831a158972412497a8C36E